### PR TITLE
Alpine based image

### PR DESCRIPTION
**Pull request by dpshekhawat**
### Description
This PR will change the base image to python-alpine to make the image lightweight. 

**Intended Outcome:**
A lightweight image with Python3 and OpenCV for machine learning model.

**Requirements**
> - [x] Python 3
> - [x] OpenCV 4.6

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,57 @@
...
+  -D BUILD_opencv_python2=NO \
+  -D BUILD_opencv_python3=ON \
+  -D PYTHON3_EXECUTABLE=/usr/local/bin/python \
+  -D PYTHON3_INCLUDE_DIR=/usr/local/include/python3.7m/ \
+  -D PYTHON3_LIBRARY=/usr/local/lib/libpython3.so \
+  -D PYTHON_LIBRARY=/usr/local/lib/libpython3.so \
+  -D PYTHON3_PACKAGES_PATH=/usr/local/lib/python3.7/site-packages/ \
+  -D PYTHON3_NUMPY_INCLUDE_DIRS=/usr/local/lib/python3.7/site-packages/numpy/core/include/ \
+  .. && \
+  make VERBOSE=1 && \
```
Comment (pkliczewski) [inline Dockerfile]: having `VERBOSE=1` for development is great. Do we want to commit it?

**Comment by pkliczewski** [comment]
image building with your dockerfile fails for me

File: Dockerfile (hunk 1 above)
Comment (dpshekhawat) [reply Dockerfile]: it is temporary will remove it before merging it to main.

**Comment by pkliczewski** [comment]
please make sure that it works. I checked and it failed for me. It would be good if you could compare it with the one I sent you

**Review by pkliczewski — APPROVED** [review]
Great work!

**Pull request merged**